scripts/install-worker.sh

```diff
@@ -48,13 +48,6 @@ else
   exit 1
 fi
 
-################################################################################
-### Utilities ##################################################################
-################################################################################
-
-sudo chmod -R a+x $TEMPLATE_DIR/bin/
-sudo mv $TEMPLATE_DIR/bin/* /usr/bin/
-
```

Comment:
I installed these in a new provisioner so they're available during `upgrade_kernel.sh`.